notification-server/src/google_cloud.rs

```diff
@@ -397,7 +397,7 @@ mod tests {
                     "amount": "200",
                     "type": "cis2-tx",
                     "token_id": "ffffff",
-                    "contract_address": {"index": 3, "subindex": 0},
+                    "contract_address": "{\"index\":3,\"subindex\":0}",
```

Comment:
Lame... but if that's what it has to be, I guess there's no way around it 🤷

Reply:
Wasn't it you who suggested using JSON for contract address? :)

Reply:
yes but I would assume the whole request being JSON would be sufficient 😛